lib/memcache/connection.ex

```diff
@@ -47,6 +47,10 @@ defmodule Memcache.Connection do
     to be used in case of connection failure. Defaults to `500`
   * `:backoff_max` - (integer) maximum allowed interval between two
     connection attempt. Defaults to `30_000`
+  * `:timeout` - (integer) connection timeout (in milliseconds) at
```

Comment:
could we call it `connect_timeout` to differentiate with send_timeout etc

Reply:
Sure, yeah, makes sense 👍